perf/benchmark/security/generate_policies/generate_policies.go

```diff
@@ -185,14 +210,111 @@ func generatePeerAuthentication(policyData SecurityPolicy, policyHeader *MyPolic
 	return yaml, nil
 }
 
+func generateToken(policyData SecurityPolicy, privateKey *rsa.PrivateKey) (string, error) {
+	issuer := fmt.Sprintf("issuer-%d", policyData.RequestAuthN.NumJwks)
+	if policyData.RequestAuthN.TokenIssuer != "" {
+		issuer = policyData.RequestAuthN.TokenIssuer
+	}
+	token := jwt.NewWithClaims(jwt.SigningMethodRS256, jwt.MapClaims{
+		"iss": issuer,
+		"sub": "subject",
+	})
+	if policyData.RequestAuthN.InvalidToken {
+		newPrivateKey, err := rsa.GenerateKey(rand.Reader, 2048)
+		if err != nil {
+			return "", err
+		}
+		privateKey = newPrivateKey
+	}
+	tokenString, err := token.SignedString(privateKey)
+	if err != nil {
+		return "", err
+	}
+	return tokenString, nil
+}
+
+func generateJwksBytes(privateKey *rsa.PrivateKey) ([]byte, error) {
+	jwks := &Jwks{
+		Keys: []*Jwk{
+			{
+				E:   base64.URLEncoding.EncodeToString(big.NewInt(int64(privateKey.PublicKey.E)).Bytes()),
```

Comment:
why do we need big.NewInt here?

Reply:
I converted it into a big.NewInt so that I could use the .Bytes() method. I could have used []byte(strconv.Itoa(E)) but this incorrectly decodes the int